meerk40t/grbl/parser.py

```diff
@@ -258,10 +253,15 @@ def parse(self, data, elements):
         for d in data:
             if isinstance(d, (bytes, bytearray)):
                 d = d.decode("utf-8")
-            self.process(d)
+            # Lets split lines...
+            splitted_lines = d.splitlines()
+            for singleline in splitted_lines:
+                self.process(singleline)
+        # We need to add a matrix to scale grbl coordinates ?!
         elements.elem_branch.add(
-            type="elem path", path=plotclass.path, stroke=Color("blue")
+            type="elem path", path=abs(plotclass.path), stroke=Color("blue"), stroke_width=UNITS_PER_PIXEL,
```

Comment:
This should probably use `stroke_width=context.elements.default_strokewidth`

Since that exists much as `default_stroke` and `default_fill` exist.